CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1] - 2019-06-14
+
+### Changed
```

Comment:
```suggestion
### Fixed
```

Reply:
Suggestion applied and agree with this convention.

To explain myself, https://keepachangelog.com/en/1.0.0/ examples are misleading, top ones coupled fixes under `Changed` and initially I just saw those examples; now that you suggested change I checked once more and found there are examples further below which have fixes under dedicated `Fixed` paragraph in changelog.

Reply:
@kopiczko PTAL

Reply:
Yeah we have to get used to that.

Reply:
Still "best" standard I have found. Feel free to suggest something more structured if you have something.